src/screens/Ranking/GroupRanking/GroupRanking.jsx

```diff
@@ -25,6 +27,10 @@ const GroupRanking = ({ name, members }) => {
     [opponents],
   )
 
+  const teams = useTeams()
+
+  const imgUrl = memoize(() => require(`../../../assets/icons/mask6.png`))
```

Comment:
Si url constante, importe là au dessus du fichier pour éviter de le faire à chaque rendu.

```suggestion
import imgUrl from '../../../assets/icons/mask6.png'
```